Dangerfile

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require './fastlane/Constants/Constants'
+require './fastlane/Constants/Constant'
```

Comment:
Does this work? It's a Swift file, while this Dangerfile contains Ruby code.

Please try it by generating a project and create a pull request and update the POW

Reply:
You're correct, this issue is not simple as I expected, I changed approach and will create another PR